_includes/exhibits.js

```diff
@@ -1957,8 +1939,32 @@ HashState.prototype = {
       const g = index_name(this.cgs, group.Name);
       this.addGroup(group, g, 'channel-groups', false);
     }, this);
+
+    const cgs_multi = this.cgs.filter(group => {
+      return group.Channels.length > 1;
+    });
+    const cgs_single = this.cgs.filter(group => {
+      return group.Channels.length == 1;
+    });
+    const cg_legend = document.getElementById('channel-groups-legend');
+    if (cgs_multi.length > 0) {
+      h = document.createElement('h6');
+      h.innerText = 'Channel Groups:'
+      h.className = 'm-1'
+      cg_legend.appendChild(h);
+    }
     // Add all channel groups to legend
-    this.cgs.forEach(function(group) {
+    cgs_multi.forEach(function(group) {
+      const g = index_name(this.cgs, group.Name);
+      this.addGroup(group, g, 'channel-groups-legend', true);
+    }, this);
+    if (cgs_single.length > 0) {
+      h = document.createElement('h6');
```

Comment:
This code is duplicated so I think it could be moved into a function which sets the proper title (Channels vs Channel groups).